tools/lsp/ui/resizer.slint

```diff
@@ -6,8 +6,17 @@ global ResizeState {
 }
 
 component ResizeHandle inherits Rectangle {
-    callback resize(/* width */ length, /* height */ length);
     in property <MouseCursor> mouse-cursor;
+    out property <bool> resizing <=> ta.pressed;
+
+    callback resize(/* width */ length, /* height */ length, /* done? */ bool);
+    callback resize-done(/* width */ length, /* height */ length);
+
+    // Internal!
+    in-out property <length> new-width;
+    in-out property <length> new-height;
+    in-out property <length> new-x;
+    in-out property <length> new-y;
```

Comment:
if it's internal, it should be private.  But i don't really understand why you need these